### PR TITLE
YLP-2484 Removing setting level column

### PR DESCRIPTION
**Pull request by fatb38**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
 ## 3.0.5-beta - 2023-07-12
+### Engineering use
```
Comment (TimCanu) [inline CHANGELOG.md:6]: **❔ question:** ‏Is it really an Engineering use thing? This is not a refactor or something that the patient won't see. It looks like a feature, don't you think?

Comment (fatb38) [reply CHANGELOG.md:6]: I had a doubt about that, ok for a feature ;)

**Pull request merged**